src/cpp/shared/game/game.cpp

```diff
@@ -1043,6 +1031,13 @@ int philAI::EvaluateHeroEvent(int a1, int a2, int a3, int a4, int *a5) {
 	return EvaluateHeroEvent_orig(a1, a2, a3, a4, a5);
 }
 
+int philAI::ValueOfEventAtPosition(int x, int y, int a2, int *a3) {
+  if(gpAdvManager->GetCell(x, y)->getLocationType() == LOCATION_SHIPYARD)
+    return 0; // ignore shipyard objects
```

Comment:
I'm curious: Did you get the infamous "Tell Phil" popup before you fixed this?

Reply:
Oh hey, I see you already posted that to the Trello card. Called it!

I got this one way back in 2010 or 2011, back when the project was just getting started.